snowflake-integration-grant-all/module_test.go

```diff
@@ -15,19 +15,38 @@ import (
 )
 
 const vars string = "{\"integration_name\":\"integration\",\"roles\":[\"role_a\",\"role_b\",\"role_c\"]}"
+const onFutureVars string = ""
 
 func TestModule(t *testing.T) {
 	test := tftest.Test{
-		// just run init, swtich to Plan or Apply when you can
-		Mode: tftest.Init,
+		Mode: tftest.Plan,
 
 		Setup: func(t *testing.T) *terraform.Options {
-			opts := tftest.Options(
-				tftest.DefaultRegion,
-				mustJson(vars),
-			)
-			opts.TerraformDir = "."
+			opts := &terraform.Options{
+				TerraformDir: ".",
+				EnvVars: map[string]string{},
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File is not `gofmt`-ed with `-s` (gofmt)